app/controllers/report_controller.rb

```diff
@@ -0,0 +1,13 @@
+class ReportController < ApplicationController
+  def month_report
+    @current_user = current_user
+    @transactions = FinancesUser.includes(:finance).by_user_id(@current_user.id)
+
+    income_transactions = @transactions.by_status(true)
+    expense_transactions = @transactions.by_status(false)
```

Comment:
```suggestion
    expense_transactions = @transactions - expense_transactions
```
đỡ dc 1 câu truy vấn nè

Reply:
ActiveRecord cho phép làm hẳn phép trừ :dancer: omg. Vâng ạ.